services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -91,10 +90,10 @@ async def _apply_observation_cycle(
         )
 
     try:
-        async with timeout(
-            dynamic_services_settings.DYNAMIC_SCHEDULER.DIRECTOR_V2_DYNAMIC_SCHEDULER_MAX_STATUS_API_DURATION
-        ):
-            await update_dynamic_sidecar_health(app, scheduler_data)
+        await asyncio.wait_for(
+            update_dynamic_sidecar_health(app, scheduler_data),
+            timeout=dynamic_services_settings.DYNAMIC_SCHEDULER.DIRECTOR_V2_DYNAMIC_SCHEDULER_MAX_STATUS_API_DURATION,
+        )
```

Comment:
You could totally bypass this timeout here. Or does the same issue not happen any longer when using `asyncio.wait_for`?
I would be in favour of removing it.

Reply:
I see from the test that issue is no longer present. Still please remove this timeout, since it is properly handled by the API already.